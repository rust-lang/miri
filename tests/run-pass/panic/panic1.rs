// rustc-env: RUST_BACKTRACE=1
// compile-flags: -Zmiri-disable-isolation
// normalize-stderr-test "at .*/(rust[^/]*|checkout)/library/.*" -> "at RUSTLIB/$$FILE:LL:COL"
// normalize-stderr-test "::<.*>" -> ""

fn main() {
    std::panic!("panicking from libstd");
}
