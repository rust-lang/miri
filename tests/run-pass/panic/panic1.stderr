thread 'main' panicked at 'panicking from libstd', $DIR/panic1.rs:7:5
stack backtrace:
   0: std::rt::begin_panic
             at RUSTLIB/$FILE:LL:COL
   1: main
             at $DIR/panic1.rs:7:5
   2: <fn() as std::ops::FnOnce<()>>::call_once - shim(fn())
             at RUSTLIB/$FILE:LL:COL
   3: std::rt::lang_start::{closure#0}
             at RUSTLIB/$FILE:LL:COL
   4: std::ops::function::impls::call_once
             at RUSTLIB/$FILE:LL:COL
   5: std::panicking::r#try::do_call
             at RUSTLIB/$FILE:LL:COL
   6: std::panicking::r#try
             at RUSTLIB/$FILE:LL:COL
   7: std::panic::catch_unwind
             at RUSTLIB/$FILE:LL:COL
   8: std::rt::lang_start_internal::{closure#2}
             at RUSTLIB/$FILE:LL:COL
   9: std::panicking::r#try::do_call
             at RUSTLIB/$FILE:LL:COL
  10: std::panicking::r#try
             at RUSTLIB/$FILE:LL:COL
  11: std::panic::catch_unwind
             at RUSTLIB/$FILE:LL:COL
  12: std::rt::lang_start_internal
             at RUSTLIB/$FILE:LL:COL
  13: std::rt::lang_start
             at RUSTLIB/$FILE:LL:COL
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
